ldms/man/ldmsctl.man

```diff
@@ -469,6 +469,16 @@ The updater name. If none is given, the statuses of all updaters are
 reported.
 .RE
 
+.SS Query the updater match list
+.BR updtr_match_list
+attr=<value>
+.RS
+.TP
+.BI [name " name"]
+.br
+The updater name. If none is given, the match list of all updaters are reported.
```

Comment:
Also, maybe provide an example request with output in the man page?

Reply:
Sure, I'll add a couple of examples.